content/applications/inventory_and_mrp/inventory/management/products/reordering_rules.rst

```diff
@@ -0,0 +1,126 @@
+================
+Reordering rules
+================
+
+Reordering rules are used to keep forecasted stock levels above a certain threshold without
+exceeding an upper limit. This is accomplished by specifying a minimum quantity that stock should
+not fall below and a maximum quantity that stock will be replenished up to.
+
+Reordering rules can be configured for each product based on the route used to replenish it. If a
+product uses the *Buy* route, then a Request for Quotation (RFQ) will be created when the reordering
+rule is triggered. If a product uses the *Manufacture* route, then a Manufacturing Order (MO) will
+be created instead. This will be the case for whichever replenishment route is selected.
+
+Configure products for reordering rules
+=======================================
+
+In order to use reordering rules for a product, it must first be correctly configured. Begin by
+navigating to :guilabel:`Inventory --> Products --> Products`, then select an existing product or
+create a new one by clicking :guilabel:`Create`.
+
+First, on the :guilabel:`General Information` tab, make sure that the :guilabel:`Product Type` is
+set to :guilabel:`Storable Product`. This is necessary because Odoo only tracks stock quantities for
+storable products, and this number is used to trigger reordering rules.
+
+.. image:: reordering_rules/product-type.png
+   :align: center
+   :alt: Set the Product Type as Storable.
+
+Next, click on the :guilabel:`Inventory` tab and select one or more routes from the
+:guilabel:`Routes` section. Doing so tells Odoo which route to use to replenish the product.
+
+.. image:: reordering_rules/select-routes.png
+   :align: center
+   :alt: Select one or more routes on the Inventory tab.
+
+If the product will be reordered using the :guilabel:`Buy` route, confirm that the :guilabel:`Can be
+Purchased` checkbox is enabled under the product name. This will make the :guilabel:`Purchase` tab
+appear. Click on the :guilabel:`Purchase` tab and specify at least one vendor and the price that
+they sell the product for, so that Odoo knows which company the product should be purchased from.
+
+.. image:: reordering_rules/specify-vendor.png
+   :align: center
+   :alt: Specify a vendor and price on the Purchase tab.
+
+If the product will be replenished using the :guilabel:`Manufacture` route, it needs to have at
+least one Bill of Materials (BoM) associated with it. This is necessary because Odoo will only
+create Manufacturing Orders for products with a :abbr:`BoM (Bill of Materials)`.
```

Comment:
```suggestion
least one Bill of Materials (BoM) associated with it. This is necessary because Odoo only creates
Manufacturing Orders for products with a :abbr:`BoM (Bill of Materials)`.
```